### PR TITLE
Update @sxzz links

### PR DESCRIPTION
**Pull request by sxzz**

### CODE REVIEW

**Review by LinuxSuRen — COMMENTED** [review]
LGTM

Actually, I'm thinking about creating a profile for each host and guest. So, people could update their information there.

For example, [archives/tisonkun.yaml](https://github.com/opensource-f2f/episode/blob/master/archives/tisonkun.yaml) is the profile of @tisonkun 

Currently, we still need to keep it to be a simple string array due to [this template](https://github.com/opensource-f2f/episode/blob/fbb0b26df9bfaa3cf9e8c705a5d4e22bdff40362/README.tpl#L14) cannot parse a complex object array.

**Comment by sxzz** [comment]
I see.

**Comment by sxzz** [comment]
@LinuxSuRen I just rename my name to Chinese only, it that ok?

File: README.md [hunk 1]
```diff
@@ -85,7 +85,7 @@
 * satohchen
 * shinyzhu
 * suave
-* sxzz
+* 三咲智子
```
Comment (LinuxSuRen) [inline README.md:88]: We don't need to update `README.md`. Instead, it is generated by `README.tpl`. I prefer to add a file `sxzz.yaml` into https://github.com/opensource-f2f/episode/tree/master/archives

Then, you could add your Chinese name like this:
```yaml
name: 三咲智子
```

Of course, we need to support to show it at https://github.com/opensource-f2f/website

**Comment by LinuxSuRen** [comment]
hi @sxzz , see also the example https://github.com/opensource-f2f/episode/pull/172

**Comment by sxzz** [comment]
@LinuxSuRen Done.

**Review by LinuxSuRen — APPROVED** [review]

**Pull request merged**